style.css

```diff
@@ -79,23 +74,274 @@ p {
   color: #7f8cff;
   text-decoration: none;
+  margin-left: 240%;
+}
+
+.works {
+  display: grid;
+  width: 100%;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  background-color: #e5e5e5;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+
+  /* border-right: 15%; */
+  max-width: 100%;
+  height: auto;
+
+  /* gap: 12px; */
+
+  /* border: 5px solid red; */
+  background: #fff;
+  border-radius: 16px;
+}
+
+.img-padding {
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+
+  /* align-items:; */
+}
+
+.company {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-left: 5%;
+  padding-bottom: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 32px;
+}
+
+.exp {
+  padding-left: 5%;
+  padding-bottom: 12px;
+}
+
+.exp li {
+  display: inline-block;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.exp-text {
+  color: #7a869a;
+}
+
+#job-info {
+  margin: 0%;
+  padding-left: 5%;
+  padding-bottom: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.skill {
+  padding-left: 4%;
+  display: flex;
+  flex-direction: row;
+  padding-bottom: 12px;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.skill li {
+  align-items: center;
+  padding: 4px 12px;
+  margin: 4px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.project-button {
+  padding-left: 5%;
+  padding-bottom: 12px;
+}
+
+.project-button button {
+  width: 126px;
+  height: 48px;
+  background-color: transparent;
+  border-color: #6070ff;
+  color: #396df2;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.skill li button {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-color: transparent;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.project-button button:hover {
+  background-color: lightblue;
+  cursor: pointer;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  margin-top: 50px;
+  background-color: white;
+  border-radius: 0 50px 0 0;
+}
+
+.sct-1 > h2 {
+  padding: 20px;
+  margin-top: 50px;
+}
+
+.description {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 13px;
+  margin-right: 20px;
+  margin-bottom: 13px;
+  padding-left: 7.5%;
+}
+
+.contacts {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-left: 9%;
+  margin-top: 5px;
+  margin-bottom: 10px;
+  padding-left: 1%;
+  padding-bottom: 12px;
+  color: #7f8cff;
+}
+
+.link {
+  padding: 3px;
+  color: #505f79;
+}
+
+.resume {
+  width: 138px;
+  height: 34px;
+  margin-left: 40px;
+  background-color: transparent;
+  border-color: #6070ff;
+  color: #396df2;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  margin-bottom: 33px;
+}
+
```

Comment:
- [OPTIOBAL] Very good job with the button 👌 Please consider using the same height for button as given in the Figma design guidelines to get a design close to Figma guidelines as much as possible ✨

Your design 🔽 | Figma guidelines 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/229306062-53130369-4008-4c68-98b1-d809786b1cbc.png) | ![image](https://user-images.githubusercontent.com/106119105/229306076-4c320a7a-bb6d-4438-8842-6fe9343f56b5.png)

Reply:
@KanzaTahreem
Ohh!! Thanks a lot for your great effort. Thanks for the review. I am having a bit problem changing padding and margin inside flexbox. I will go through freecodecamp tomorrow Inn Shaa Allah.

Reply:
@KanzaTahreem Done.

Reply:
Excellent job @Zafron047 💪💪 Keep going. Let me know if you need any help. I wish you success 🥇🥇